process-compose/suzuka-full-node/process-compose.yml

```diff
@@ -32,7 +32,7 @@ processes:
 
   suzuka-full-node:
     command: |
-      ./target/$CARGO_PROFILE/suzuka-full-node
+      RUST_LOG=debug ./target/$CARGO_PROFILE/suzuka-full-node
     depends_on:
```

Comment:
The node should probably depend on `deploy-mcr` to make sure that the MCR configuration has been written.
But `deploy-mcr` is not defined in this file, only in the `anvil` overlay. 🤔